camelot/view/action_steps/crud.py

```diff
@@ -108,14 +115,55 @@ class SetColumns(ActionStep):
     
     def __init__(self, static_field_attributes):
         self.static_field_attributes = static_field_attributes
+        self.column_attributes = []
+        for fa in static_field_attributes:
+            attrs = {}
+            if issubclass(fa['delegate'], (delegates.ComboBoxDelegate, delegates.Many2OneDelegate,
+                                           delegates.FileDelegate)):
+                attrs = self.filter_attributes(fa, ['action_routes'])
+            elif issubclass(fa['delegate'], delegates.DateDelegate):
+                attrs = self.filter_attributes(fa, ['nullable', 'validator'])
+                if issubclass(fa['delegate'], delegates.DateTimeDelegate):
+                    if 'editable' in fa:
+                        attrs['editable'] = fa['editable']
+            elif issubclass(fa['delegate'], delegates.DbImageDelegate):
+                attrs = self.filter_attributes(fa, ['preview_width', 'preview_height', 'max_size'])
+            elif issubclass(fa['delegate'], delegates.FloatDelegate):
+                attrs = self.filter_attributes(fa, ['calculator', 'decimal', 'action_routes'])
```

Comment:
we hebben geen float velden, enkel decimal velden, dus dat mag in principe ook hardcoded zijn